tests/shell/check_kdb_internal_check.sh

```diff
@@ -33,6 +33,23 @@ do
 		;;
 	esac
 
+	# The following checks fail on an ASAN enabled build
+	# See also: https://github.com/ElektraInitiative/libelektra/pull/1963
```

Comment:
please document in doc/todo/TESTING

Reply:
done in 90a66087419867be2750445fde8636a6ebf57f11